CHANGELOG.md

```diff
@@ -9,6 +9,12 @@
 
 - Fix an issue with `AnchoredThreads` component not working correctly on certain
   React versions.
+- Fixes an issue where React components don’t update when
+  `shouldRerenderOnTransaction`: false is set
+
+### `@liveblocks/yjs`
+
+- Adds experimental offline support for `LiveblocksYjsProvider`.
```

Comment:
The diff of this PR seems to suggest that experimental offline support already exists in `LiveblocksYjsProvider`, but that we're just exposing it to Tiptap here now. Should that be in the changelog instead?

Reply:
This is because I screwed this branch up.

I'm going to re-make it because this change actually didn't even work, the tiptap option isn't always the right value during startup and the option must be set during the constructor. I will make a new PR with a new approach.